File: examples/startup-imports.js
#!/usr/bin/env node

// Node with common imports - still fast!
const fs = require('fs');
const path = require('path');
const os = require('os');
const child_process = require('child_process');
const readline = require('readline');

const args = process.argv.slice(2);
console.log(`Hello from Node with imports! Args: ${args.join(', ') || 'none'}`);
process.exit(0);

File: startup-imports.py
#!/usr/bin/env python3

# Python with common imports - much slower!
import json
import os
import sys
import subprocess
import argparse
import logging
import datetime

args = sys.argv[1:]
print(f"Hello from Python with imports! Args: {', '.join(args) if args else 'none'}")
sys.exit(0)